src/struct_info_internal.json

```diff
@@ -24,6 +24,12 @@
           "SIGCANCEL"
         ]
     },
+    {
```

Comment:
is this related to the preJS preprocessing?

Reply:
This is needed by `threadprofiler.js` which gets included as pre-js.   It makes use of this to fix it `--thread-profiler` under wasm64.  I forgot to mention that in the description.  I will add that now.